build.properties.dist

```diff
@@ -42,8 +42,9 @@ drupal.demo.users = administrator, moderator
 # Paths to executables.
 composer.bin = ${project.basedir}/vendor/bin/composer
 drush.bin = ${project.basedir}/vendor/bin/drush
-phing.bin = ${project.basedir}/vendor/bin/phing
 isql.bin = /usr/bin/isql-vt
+phing.bin = ${project.basedir}/vendor/bin/phing
+solr.bin = ${solr.dir}/bin/solr
```

Comment:
I would change this to the symlink in `${project.basedir}/vendor/bin/solr`. This would make it consistent with the other executables.